src/scss/components/checkbox/mixins/_hxCheckboxControl.scss

```diff
@@ -9,29 +9,31 @@
 
     &::-ms-check {
       border-color: $blue-700;
-      border-color: var(--hxCheckbox-focus-bordercolor, $blue-700);
+      border-color: $rax-component-checkbox-focus-border-color;
     }
   }
   @else if $pseudo-state == hover {
     color: $blue-700;
-    color: var(--hxCheckbox-hover-color, $blue-700);
+    color: $rax-component-checkbox-hover-border-color;
 
     &::-ms-check {
       background-color: $gray-0;
-      border-color: $blue-700;
+      background-color: $rax-component-checkbox-hover-background-color;
+      border-color: $rax-component-checkbox-hover-border-color;
       color: $blue-700;
+      color: $rax-component-checkbox-hover-border-color;
     }
   }
   @else {
     color: $gray-500;
-    color: var(--hxCheckbox-color, $gray-500);
+    color: $rax-component-checkbox-default-border-color;
 
     &::-ms-check {
       background-color: $gray-0;
-      background-color: var(--hxCheckbox-bgcolor, $gray-0);
+      background-color: $rax-component-checkbox-default-background-color;
       border-color: currentColor;
-      border-style: solid;
-      border-width: 1px;
+      border-style: $rax-component-input-base-border-style;
```

Comment:
Let's not mix component design tokens. We should annotate missing definitions instead.